### PR TITLE
Adding upgrade job for agent installed cluster

### PR DESCRIPTION
**Pull request by jhou1**

### CODE REVIEW

**Comment by jhou1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.12-baremetal-ha-agent-ipv4-static-disconnected-p1-f7

**Comment by jhou1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.12-baremetal-ha-agent-ipv4-static-disconnected-p1-f7

**Comment by jhou1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.12-baremetal-ha-agent-ipv4-static-disconnected-p1-f7

**Comment by jhou1** [comment]
Done [testing](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/38097/rehearse-38097-periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.12-baremetal-ha-agent-ipv4-static-disconnected-p1-f7/1644155601460137984).

/pj-rehearse ack

/assign @liangxia PTAL, thank you.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml [hunk 1]
```diff
@@ -431,6 +431,24 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-upi
+- as: baremetal-ha-agent-ipv4-static-disconnected-p1-f7
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml:434]: Do we need `UPGRADE_PRE/CHECK_RUN_TAGS` for this ? I can see aws test cases are executed in the log.

**Pull request closed without merging**